### ISSUE TITLE
[meta] Tracking issue for internationalization

### ISSUE THREAD

**Issue by Manishearth** [rust-lang/www.rust-lang.org#799]
This is the tracking issue for internationalization of the website

 - [x] Basic implementation https://github.com/rust-lang/www.rust-lang.org/pull/792
 - [ ] Add translation strings everywhere (help wanted! https://github.com/rust-lang/www.rust-lang.org/issues/798)
 - [ ] Translate all the things

**Comment by Julian-Wollersberger** [comment]
Is there a place for language-specific diskussions?
For example, in German "you" can either be translated to an informal "du" or a formal "Sie". It is important to get this consistent across the site. 
Some sort of discussion and resulting guidelines would be helpful for all languages.

**Comment by skade** [comment]
@Julian-Wollersberger We're thinking about a place to set those up, in the meantime, feel free to use #website in the rust-lang discord for that. (Or discussion what a solution would be)

**Comment by merkrafter** [comment]
@skade Is there any solution in place yet? I also would like to provide german translations, but obviously don't want to work on something that somebody else has done in the background already.

**Comment by lupomontero** [comment]
Hi there,

First of all: Thanks for all the great work!

I'd like to help out with the spanish translation... what's the state of this?

I noticed there is no spanish _option_ in the language selector in the top right hand corner of the website, so I started searching to see if I could help with this translation.

I followed the guidelines in [TRANSLATIONS.md](https://github.com/rust-lang/www.rust-lang.org/blob/master/TRANSLATIONS.md) and created an account on [pontoon](https://pontoon.rust-lang.org/), but in there all strings seem to have been translated and reviewed already... :thinking: 

I also forked and cloned this repo to dig around and found an _almost_ complete spanish translation in the [`locales/es` directory in the `master` branch](https://github.com/rust-lang/www.rust-lang.org/tree/master/locales/es). Judging by the commit logs, the most recent changes to these files seem to come from pontoon, ...

I had no trouble running the site locally (`master` branch) and _enabling_ the spanish translation in
[here](https://github.com/rust-lang/www.rust-lang.org/blob/master/src/i18n.rs#L35) :star: 

But I then noticed there were a few things missing in the spanish translation (ie: _thanks panel_ in homepage, privacy page, sponsors page), ... how can I add these? Do these files/strings need to be added in pontoon or should I send a PR directly to this repo, and if so what branch should it point to?
